target/linux/ar71xx/files/arch/mips/ath79/mach-tl-wr1041n-v2.c

```diff
@@ -90,7 +106,10 @@ static struct ar8327_platform_data db120_ar8327_data = {
 		.duplex = 1,
 		.txpause = 1,
 		.rxpause = 1,
-	}
+	},
+	.led_cfg = &wr1041n_v2_ar8327_led_cfg,
+	.num_leds = ARRAY_SIZE(tl_wr1041n_leds_ar8327),
+	.leds = tl_wr1041n_leds_ar8327
```

Comment:
I think the indention is wrong here

Reply:
Sorry, I don't get it quite well, the indention seems to be OK.